src/components/experimental/Snackbar/docs/Snackbar.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { Snackbar } from '../Snackbar';
+import { XCrossIcon } from '../../../../icons';
+import { getSemanticValue } from '../../../../essentials/experimental/cssVariables';
+
+const meta: Meta = {
+    title: 'Experimental/Components/Snackbar',
+    component: Snackbar,
+    parameters: {
+        layout: 'centered'
+    },
+    argTypes: {},
+    args: {
+        children: 'Booking successfully cancelled'
+    }
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Snackbar>;
+
+export const Default: Story = {};
+
+export const WithDismissIcon: Story = {
+    args: {
+        children: (
+            <>
+                Booking successfully cancelled
+                <XCrossIcon
+                    cursor="pointer"
+                    size={30}
```

Comment:
Size should be 24
![Uploading Screenshot 2024-11-27 at 12.37.38.png…]()

Reply:
Should this button be part of the component? This will be a common case

Reply:
Added the dismiss button as part of the component with the correct size. Also the container paddings are corrected. Thank you!